frontend/src/app/testQueue/QueueItem.tsx

```diff
@@ -229,6 +232,10 @@ const QueueItem = ({
     dateTestedProp || undefined
   );
 
+  const organization = useSelector(
```

Comment:
**suggestion:** Can this be strictly typed instead of using **as**? 

```suggestion
  const organization = useSelector<RootState, Organization>((
```

Reply:
See eca597f.

Also, I had been looking for something like `RootState` for a while so this is great. The shape of the Redux store had been a little opaque to me until now.